src/test/javascript/e2e/scenarios.js

```diff
@@ -0,0 +1,27 @@
+describe('appIdea', function() {
+
+	browser.get('/');
+
+	it('should filter results ideas list', function() {
+		// Find the all and get second (and only) button on the page and click it
+		element.all(by.css("a[type='button']")).get(1).click();
```

Comment:
if it is possible it would be better to add id or name for specific buttons to make tests more readable